src/xlsx/functions/charts.ts

```diff
@@ -83,7 +84,9 @@ export function createChart(
       plot = addScatterChart(chart.data);
       break;
     case "pie":
-      plot = addDoughnutChart(chart.data, chartSheetIndex, data, { holeSize: 0 });
+      plot = addDoughnutChart(chart.data, chartSheetIndex, data, {
+        holeSize: chart.data.isDoughnut ? 50 : 0,
+      });
```

Comment:
since we already give `chart.data` to `addDoughnutChart`, couldn't we do the holeSize thing inside `addDoughnutChart` ?